native_client/README.md

```diff
@@ -67,15 +67,19 @@ ln -s ../DeepSpeech/native_client ./
 
 ## Building
 
-Before building the DeepSpeech client libraries, you will need to prepare your environment to configure and build TensorFlow. Follow the [instructions](https://www.tensorflow.org/install/install_sources) on the TensorFlow site for your platform, up to the end of 'Configure the installation'.
+Before building the DeepSpeech client libraries, you will need to prepare your environment to configure and build TensorFlow. 
+Preferably, checkout the version of tensorflow which is currently supported by DeepSpeech (see requirements.txt), and use bazel version 0.10.0 (notably, linker errors have been reported with v0.12.0). 
```

Comment:
Try to avoid specifying a version, because we will forget to update it. We can remove the part "(notably, linker errors have been reported with v0.12.0)"